addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/NewDevicesParser.java

```diff
@@ -34,7 +34,7 @@
                 boolean isDevice = !StringUtils.contains(address, ":")
                         && !StringUtils.startsWithIgnoreCase(address, "BidCos");
                 if (isDevice) {
-                    adresses.add(address);
+                    adresses.add(getAddress(address));
```

Comment:
Maybe `getDeviceAddress` would be a better name for the `getAddress` method? It's strange to have an address as input and also get an address as output unless they are different kind of addresses.

Reply:
... or something to indicate that it sanitizes it :-) 

Because a quick look suggest its same address but then without illegal characters? What would be sensible?